### PR TITLE
Clean up some "unused" warnings

### PR DESCRIPTION
**Pull request by keveleigh**
## Overview

Cleans up a couple examples of fields or methods not being properly scoped by `#if` blocks and appearing as "unused" in certain build configurations.

Also improved some docs in the files I was updating.

### CODE REVIEW

File: Assets/MRTK/Providers/WindowsMixedReality/XR2018/WindowsMixedRealityEyeGazeDataProvider.cs [hunk 1]
```diff
@@ -259,9 +262,6 @@ private Ray SmoothGaze(Ray? newGaze)
                     isSaccading = true;
                 }
 
-                Vector3 v1 = oldGaze.Value.origin + oldGaze.Value.direction;
-                Vector3 v2 = newGaze.Value.origin + newGaze.Value.direction;
-
```
Comment (CDiaz-MS) [inline Assets/MRTK/Providers/WindowsMixedReality/XR2018/WindowsMixedRealityEyeGazeDataProvider.cs:264]: Were these values not used anywhere and that is why they were removed?

Comment (keveleigh) [reply Assets/MRTK/Providers/WindowsMixedReality/XR2018/WindowsMixedRealityEyeGazeDataProvider.cs:264]: Yep! Oddly, they were calculated but the resulting vectors weren't actually used. I tried to walk through the history a bit, but couldn't find a point where they were ever used.

**Pull request merged**